src/modules/ProjectTable/Row.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { SERVER } from 'constants.js';
+
+// Custom Hooks
+import useFetch from 'customHooks/useFetch';
+
+
+const Row = ({
+  title,
+  description,
+  status,
+  clientID,
+  id,
+}) => {
+  const [expand, setExpand] = useState(false);
+
+  const {
+    data: client,
+    isPending: clientPending,
+    error: clientError,
+  } = useFetch(`${SERVER}/clients?id=${clientID}`);
+
+  const {
+    data: invoice,
+    isPending: invoicePending,
+    error: invoiceError,
+  } = useFetch(`${SERVER}/invoices?projectId=${id}`);
+
+  const handleClick = () => {
+    setExpand(!expand);
+  };
+
+  return (
```

Comment:
Is it possible to refactor this into some smaller components?

Reply:
Yep, I can rebuild this into a smaller component.